src/main/webapp/js/studentProfile.js

```diff
@@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
         	$('#statusMessage').css("display", "block")
         	        .attr('class', 'alert alert-danger')
         	        .html('There seems to be a network error, please try again later');
-        	$("html, body").animate({
-        		scrollTop: 0
-        	});
+        	scrollToTop({duration: ''});
```

Comment:
I just looked at it again, it seems you're using the falsy nature of empty strings to replicate the functionality, perhaps add a comment will do or it might be confusing for future maintainers :)